.github/scripts/split-tests-by-timings.ts

```diff
@@ -19,9 +19,12 @@ function readTestResults(TEST_RESULTS_PATH: string): TestRun | undefined {
       readFileSync(TEST_RESULTS_PATH, 'utf8'),
     );
 
-    const testRun: TestRun | undefined = testRuns.find(
+    const testRun: TestRun = testRuns.find(
       (run) => run.name === testSuiteName,
-    );
+    ) || { // If the file doesn't exist, return a dummy object to do the naïve split
```

Comment:
This comment should be slightly changed, as in this case it's not a file that's not found, but a section of the JSON that's not found.
```suggestion
    ) || { // If the TestSuite is not found, return a dummy object to do the naïve split
```